README.md

```diff
@@ -1,33 +1,35 @@
-# Operate First template for repositories
+# Project Meteor
 
-Derive new repositories from this template
+Similar to how a meteoroid entering the atmosphere, becomes visible as a meteor and finally hitting the Earth's surface as a meteorite, we aim to create better visibility and impact of the AICoE team's work through project *Meteor*.
 
-List of featurese:
+AICoE is the Artificial Intelligence Center of Excellence team, part of Red Hat's Office of the CTO established to enable Analytics, Machine Learning, and AI cross-functionally and bundle forward-looking R&D. The team's focus is to apply AI to our core business and services through an Open Source driven development to enable user's needs and foster open collaboration.
 
-## License
+It is an AICoE combined team effort to provide a product for Data Scientists and other users alike where they can interact and leverage our services, tools and technology for developing various applications. The product can also help in introducing, sharing and promoting our team's capabilities with others.
 
-This template ensures new repos are created compliant with [ADR 0001](https://www.operate-first.cloud/blueprints/blueprint/docs/adr/0001-use-gpl3-as-license.md) and use GNU GPL v3 license.
+## Product
 
-## AI-CoE CI Github application
+The product is an interactive [Elyra tutorial](https://github.com/AICoE/elyra-aidevsecops-tutorial) web application that is rendered as a [JupyterBook](https://jupyterbook.org/) environment by taking as an input the user's GitHub repository (user's fork of the Elyra tutorial repository) and runs the build/deployment pipelines in the backend to render the notebook environment as the final output. You can also take a look at the product flowchart for more details [here](https://github.com/AICoE/meteor/blob/main/docs/project-flowchart.md).
```

Comment:
```suggestion
The product is an interactive [AiDevSecOps tutorial](https://github.com/AICoE/elyra-aidevsecops-tutorial) and web application that is rendered as both a static [JupyterBook](https://jupyterbook.org/) and an interactive Jupyter Lab environment by taking as an input the user's GitHub repository (specifically a fork of the tutorial repository) and runs the build/deployment pipelines in the background to render the notebook environments as the final output.
```
Can you just show the flow chart as an image here instead of linking to it?